nixos/tests/virtualbox.nix

```diff
@@ -205,96 +205,105 @@ let
     };
 
     testSubs = ''
```

Comment:
This is pretty ugly and the reason why I used the `name` attribute back then was to work around lack of serialisation. With Python however, [`json`](https://docs.python.org/3/library/json.html) is part of the kitchen sink, so we can simply `toJSON` the attributes above.

Reply:
I felt that it would be more appropriate to keep this a simple port/translation rather than start refactoring. I can change this to use a dict, though, if you prefer?

Reply:
I am generally also in favor of starting with a 1:1 translation generally. but @aszlig's suggestion is too good to not do it right now.